README.md

```diff
@@ -1,2 +1,5 @@
 # week2day1
 Project Review
+
+
+Hope this is enough to create a pull request.
```

Comment:
Well done @Yordinia, I could not find the required changes as requested by the previous reviewers😐 👇.
[REQUIRED]  From the project requirement you should use at least one inline comment. Please check the following link to see how to do [inline comment](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) as mentioned by the previous reviewer 😄.

Kindly click the link above, it contains a video of how to make an inline comment.

Reply:
Hi @Lizdev-05  I have countlessly included an inline comment in this pull request. Please check it for me before  I request additional code review